package.json

```
@@ -5,6 +5,7 @@
   "license": "MPL-2.0",
   "scripts": {
     "lint": "eslint --fix . && prettier --write **/*.ts **/{metadata,tsconfig}.json",
+    "lint:ci": "eslint .",
```

rhys do you ever listen to me
```suggestion
    "lint:ci": "eslint . && prettier **/*.ts **/{metadata,tsconfig}.json",
```